src/screens/DomainScreen/elements/RenderItem.js

```diff
@@ -60,8 +77,8 @@ const RenderItem = ({
         }
 
         if (item.latest_reactions.downvotes !== undefined) {
-          const downvotes = item.latest_reactions.downvotes.filter(
-            (vote) => vote.user_id === selfUserId
+          let downvotes = item.latest_reactions.downvotes.filter(
+            (vote) => vote.user_id === selfUserId,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the variable `downvotes` has been changed from a constant to a let variable without being reassigned. It would be better to keep it as a constant.
```diff
- let downvotes = item.latest_reactions.downvotes.filter(
+ const downvotes = item.latest_reactions.downvotes.filter(
    (vote) => vote.user_id === selfUserId,
  );
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->